crates/rspack_loader_sass/tests/fixtures/rspack_importer/expected/main.css

```diff
@@ -2,6 +2,7 @@
   color: red;
 }
 @charset "UTF-8";
+;
```

Comment:
this seems unexpected

Reply:
This always cause by https://github.com/swc-project/swc/issues/7584, the end of decl is not correct.